locations/spiders/puebloweb_pr_us.py

```diff
@@ -6,7 +6,6 @@ class PueblowebPRUSSpider(ElfsightSpider):
     name = "puebloweb_pr_us"
```

Comment:
Can be renamed to `pueblo_pr_us`. The `web` suffix is not part of the brand name and is seemingly just their web domain suffix as `pueblo.com` was seemingly already taken?

Reply:
(and file name and class name updated too)

Reply:
Can the `_us`/`US` be dropped from the name as all locations appear to be in `PR` only?